server/services/v1/database/query_runner.go

```diff
@@ -218,6 +254,14 @@ func (runner *UpdateQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 		}
 	}
 
+	if fieldOperator, ok := factory.FieldOperators[string(update.Push)]; ok {
+		// mutate if push payload needs set default values and also need to convert numeric fields from string to int64
```

Comment:
We can safely assume we don't need to set defaults inside an Array. This will make this operation less expensive.

Reply:
Okay, reverted the change. Not setting default values inside an array.